packages/message-relayer/src/exec/run.ts

```diff
@@ -14,6 +16,27 @@ const main = async () => {
   })
 
   const env = process.env
+
+  const SENTRY_DSN = config.str('use-sentry', env.SENTRY_DSN)
+  const USE_SENTRY = config.str('sentry-dsn', env.USE_SENTRY)
+  const ETH_NETWORK_NAME = config.str('eth-network-name', env.ETH_NETWORK_NAME)
+
+  let loggerOptions = {
+    name: 'Message_Relayer',
+  }
+
+  if (USE_SENTRY) {
+    const sentryOptions = {
+      release: `message-relayer@${process.env.npm_package_version}`,
+      dsn: SENTRY_DSN,
+      environment: ETH_NETWORK_NAME,
+    }
+    loggerOptions['sentryOptions'] = sentryOptions
```

Comment:
Nit: I don't often see this syntax in JS, it should be equivalent to `loggerOptions.sentryOptions = sentryOptions`. Any particular reason to use more verbose syntax?

Reply:
86ffa983